tests/unit/amo/pages/TestAddonReviewList.js

```diff
@@ -290,887 +265,793 @@ describe(__filename, () => {
-      expect(header.find(LoadingText)).toHaveLength(1);
+      // Expect 2 per placeholder review (of which there should be 4), plus
+      // 1 in the header.
+      expect(
+        within(screen.getByClassName('AddonReviewList-reviews')).getAllByRole(
+          'alert',
+        ),
+      ).toHaveLength(9);
     });
 
     describe('with pagination', () => {
       const renderWithPagination = ({
-        addon = fakeAddon,
         page = '1',
         reviews = Array(DEFAULT_API_PAGE_SIZE + 2).fill(fakeReview),
         ...otherProps
       } = {}) => {
-        _loadAddon(addon);
-        _setAddonReviews({ addon, page, reviews });
-
-        return render({
-          location: createFakeLocation({ query: { page } }),
-          ...otherProps,
-        });
-      };
-
-      const renderFooter = (root) => {
-        return shallow(
-          root.find('.AddonReviewList-reviews-listing').prop('footer'),
-        );
+        return renderWithAddonAndReviews({ page, reviews, ...otherProps });
       };
 
       it('configures a paginator with the right URL', () => {
-        const addonSlug = 'adblock-plus';
-        const addon = { ...fakeAddon, id: 8765, slug: addonSlug };
-        _loadAddon(addon);
-        const root = renderWithPagination({ addon, params: { addonSlug } });
-
-        const paginator = renderFooter(root);
+        renderWithPagination();
 
-        expect(paginator.instance()).toBeInstanceOf(Paginate);
-        expect(paginator).toHaveProp('pathname', reviewListURL({ addonSlug }));
+        expect(screen.getByRole('link', { name: 'Next' })).toHaveAttribute(
+          'href',
+          `/en-US/firefox/addon/${defaultSlug}/reviews/?page=2`,
+        );
       });
 
       it('adds UTM query parameters to the reviews URL when there are some', () => {
-        const utm_campaign = 'some-utm-campaign';
-        const location = createFakeLocation({ query: { utm_campaign } });
-        const addonSlug = 'adblock-plus';
-        const addon = { ...fakeAddon, id: 8765, slug: addonSlug };
-        _loadAddon(addon);
-
-        const root = renderWithPagination({
-          addon,
-          params: { addonSlug },
-          location,
+        const utmCampaign = 'some-utm-campaign';
+        renderWithPagination({
+          location: `${getLocation()}&utm_campaign=${utmCampaign}`,
         });
 
-        expect(renderFooter(root)).toHaveProp(
-          'pathname',
-          `${getAddonURL(addonSlug)}reviews/?utm_campaign=${utm_campaign}`,
+        expect(screen.getByRole('link', { name: 'Next' })).toHaveAttribute(
+          'href',
+          `/en-US/firefox/addon/${defaultSlug}/reviews/?utm_campaign=${utmCampaign}&page=2`,
         );
       });
 
-      it('configures a paginator with the right Link', () => {
-        const root = renderWithPagination();
-
-        expect(renderFooter(root)).toHaveProp('LinkComponent', Link);
-      });
-
       it('configures a paginator with the right review count', () => {
```

Comment:
Where does this test check the correct review count is configured?  Is it just that there is pagination rendered for 2 pages?

Reply:
Yes, if the review count was used for the `paginator` then we should see "Page 1 of 2".